src/test/util.ts

```diff
@@ -162,3 +162,6 @@ export function formatLocation(location: Location) {
 export function errorWithFile(file: string, message: string) {
   return new Error(`${relativeFilePath(file)}: ${message}`);
 }
+
+export const builtInReporters = ['list', 'line', 'dot', 'json', 'junit', 'null'] as const;
+export type BuiltInReporters = typeof builtInReporters[number];
```

Comment:
- nit: BulitInReporters -> BuiltInReporter?
- I'd move this to runner or cli, doesn't look like "util".